layouts/default.slim

```diff
@@ -3,9 +3,13 @@ html
   head
     == render '/header.*'
   body lang == (@item[:lang] || 'cn')
-    #content
-      == render '/menu.*'
-      == yield
+    .pusher
```

Comment:
Ok, it can be done somehow. I do it because the semanic.min.js keeps tell me that it's better to make the sidebar as the direct children of body tag. But it's not great problem.